### DISCUSSION TITLE
Patient metadata in WFDB files

### DISCUSSION THREAD

**Discussion by cx1111** [wfdb/wfdb-spec#6] [General]
WFDB header files support general comment lines. The original WFDB software package, and some other PhysioNet applications 

Some other formats enforce more structured patient information fields, including EDF+ and DICOM. I understand the utility of this, especially when passing around individual files for analysis that should ideally be inclusive of all the required information. But I also find it strange that waveform files are expected to contain this content. 

What do people think about whether WFDB files should have structured fields about patient metadata?

**Comment by tompollard** [comment]
@cx1111 I would really like to see more structured header information for WFDB. I think we should have a clear structure for managing metadata such as patient metadata even if fields are optional.

e.g. in the past I've seen information like patient ID, hospital admission ID, patient age, gender, etc appearing in the headers. If there is demand to include this information, we should be clear about how it is handled.

**Comment by cx1111** [comment]
Moved over to #14
